modules/running-a-training-job-by-using-the-kfto-sdk.adoc

```diff
@@ -0,0 +1,128 @@
+
+[NOTE]
+====
+The code in this procedure specifies how to run the example training job. 
+If you have the specified resources, you can run the example code without editing it.
+
+Alternatively, you can modify the example code to specify the appropriate details for your training job.
+====
+
+.Prerequisites
+
+* You can access an OpenShift cluster that has sufficient worker nodes with supported accelerators to run your training or tuning job.
+
+
+ifndef::upstream[]
+* You can access a workbench that is suitable for distributed training, as described in link:{rhoaidocshome}{default-format-url}/working_with_distributed_workloads/preparing-the-distributed-training-environment_distributed-workloads#creating-a-workbench-for-distributed-training_distributed-workloads[Creating a workbench for distributed training].
+endif::[]
+ifdef::upstream[]
+* You can access a workbench that is suitable for distributed training, as described in link:{odhdocshome}/working-with-distributed-workloads/#creating-a-workbench-for-distributed-training_distributed-workloads[Creating a workbench for distributed training].
+endif::[]
+
+* You have administrator access for the data science project.
+** If you created the project, you automatically have administrator access. 
+** If you did not create the project, your cluster administrator must give you administrator access.
+
+* You have access to a model.
+* You have access to data that you can use to train the model.
+
+ifndef::upstream[]
+* You configured the training job as described in link:{rhoaidocshome}{default-format-url}/working_with_distributed_workloads/using-the-kubeflow-sdk-to-run-distributed-training_distributed-workloads#configuring-a-training-job-by-using-the-kfto-sdk_distributed-workloads[Configuring a training job by using the KFTO SDK].
+endif::[]
+ifdef::upstream[]
+* You configured the training job as described in link:{odhdocshome}/working-with-distributed-workloads/#configuring-a-training-job-by-using-the-kfto-sdk_distributed-workloads[Configuring a training job by using the KFTO SDK].
+endif::[]
+
+
+.Procedure
+. Open the workbench, as follows:
+.. Log in to the {productname-long} web console.
+.. Click *Data science projects* and click your project.
+.. Click the *Workbenches* tab. 
+If your workbench is not already running, start the workbench.
+.. Click the *Open* link to open the IDE in a new window. 
+
+. Click *File -> Open*, and open the notebook that you used to configure the training job.
+
+. Create a cell to run the job, and add the following content:
++
+[source,subs="+quotes"]
+----
+from kubernetes import client
+from kubeflow.training import TrainingClient
+
+# Use API client for authentication (Recommended)
+client = TrainingClient(client_configuration=api_client.configuration)
+
+# Start PyTorchJob with 2 Workers and 2 GPU per Worker (e.g. multi-node, multi-worker job).
+from kubeflow.training import TrainingClient
```

Comment:
This line looks like a copy paste duplication and can be removed, import statement is already provided above.